packages/wallets/src/components/SentEmailContent/SentEmailContent.tsx

```diff
@@ -30,23 +32,28 @@ type SentEmailContentProps = {
 // NOTE: key field is not from BE or requirements, its only used for key prop
 const emailReasons = [
     {
-        content: 'The email is in your spam folder (Sometimes things get lost there).',
+        content: <Localize i18n_default_text='The email is in your spam folder (Sometimes things get lost there).' />,
```

Comment:
Cant we just use the Localize component here instead of repeating it in every content ?
` <Text lineHeight='sm' size={emailReasonsSize}>
                                        {emailReason.content}
                                    </Text>`

Reply:
@thisyahlen-deriv for the deriv-com/translations `Localize` we have to wrap the strings themselves. Wrapping dynamic content in this method will cause it to not get uploaded to crowdin

Reply:
Oh dammmnnn, aight aight